projects/packages/sync/src/modules/class-woocommerce.php

```diff
@@ -128,6 +148,8 @@ public function init_listeners( $callable ) {
 		add_action( 'woocommerce_new_order_item', $callable, 10, 4 );
 		add_action( 'woocommerce_update_order_item', $callable, 10, 4 );
 		add_action( 'woocommerce_delete_order_item', $callable, 10, 1 );
+		add_action( 'woocommerce_remove_order_items', array( $this, 'action_woocommerce_remove_order_items' ), 10, 2 );
```

Comment:
Should we move that `add_action` into the constructor with other filters?

Reply:
Moved to the constructor 👍